src/applications/claims-status/actions/index.jsx

```diff
@@ -196,7 +196,7 @@ export function getClaimsV2(poll = pollRequest) {
         dispatch({ type: FETCH_CLAIMS_ERROR });
       },
       onSuccess: response => dispatch(fetchClaimsSuccess(response)),
-      pollingInterval: window.VetsGov.pollTimeout || 1000,
+      pollingInterval: window.VetsGov.pollTimeout || 5000,
```

Comment:
Do we know that `window.VetsGov.pollTimeout` is not set?

Reply:
Yeah, I looked for that. That's used only for unit tests, I believe, so we don't have to wait for the full polling duration in those tests.